pom.xml

```diff
@@ -104,7 +104,7 @@
     <avro.version>1.8.1</avro.version>
     <slf4j.version>1.7.5</slf4j.version>
     <google.http.client.version>1.22.0</google.http.client.version>
-    <protobuf.version>3.3.0</protobuf.version>
+    <protobuf.version>3.4.0</protobuf.version>
```

Comment:
updated to 3.4.0 as i was noticing the issue with NoSuchMethodError for `AbstractMessageLite#addAll(...)` as specified in this issue https://github.com/protocolbuffers/protobuf/issues/4521